src/components/molecules/math-operation/math-operation.component.tsx

```diff
@@ -1,11 +1,10 @@
 import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
-import Label from '../../atoms/label/label.component';
-import Input from '../../atoms/input/input.component';
-import Button from '../../atoms/button/button.component';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import BaseButton from '../../atoms/button/button.component';
 import ResultDisplay from '../../atoms/result-display/result-display.component';
-import ErrorMessage from '../../atoms/error-message/error-message.component';
-import './math-operation.styles.scss'
+import { Paper, Grid, Typography, TextField, Box } from '@mui/material';
```

Comment:
Rozważ użycie reguły `eslint` służącej do sortowania importów

https://eslint.org/docs/latest/rules/sort-imports

Reply:
Czy można ustawić, żeby przy zapisie ESLint to zmieniał, czy muszę używać "npm run lint" do tego?